src/amo/components/UserProfile/index.js

```diff
@@ -60,9 +70,23 @@ export class UserProfileBase extends React.Component<Props> {
     }
   }
 
+  getEditURL() {
+    const { currentUser, user } = this.props;
+
+    invariant(user, 'user is required');
+    invariant(currentUser, 'currentUser is required');
+
+    if (currentUser.id === user.id) {
+      return `/users/edit`;
```

Comment:
Because there is no string substitution here, and because this is also referenced in a test, would it be worth making this a constant that can be imported by the test?